From 571284254d0adf76b279177599c3ddda6749b3a3 Mon Sep 17 00:00:00 2001
From: Mike Looijmans <mike.looijmans@topic.nl>
Date: Mon, 27 Jan 2014 08:52:35 +0100
Subject: [PATCH] Revert "arm: zynq: Enable the Neon instructions"

This reverts commit 05343b241c394362f14869167325367550333187.

Conflicts:

	arch/arm/cpu/armv7/zynq/cpu.c
	arch/arm/cpu/armv7/zynq/lowlevel_init.S
---
 arch/arm/cpu/armv7/zynq/Makefile        |    8 ++------
 arch/arm/cpu/armv7/zynq/cpu.c           |    4 ++++
 arch/arm/cpu/armv7/zynq/lowlevel_init.S |   26 --------------------------
 3 files changed, 6 insertions(+), 32 deletions(-)
 delete mode 100644 arch/arm/cpu/armv7/zynq/lowlevel_init.S

diff --git a/arch/arm/cpu/armv7/zynq/Makefile b/arch/arm/cpu/armv7/zynq/Makefile
index 242ca6b..1510fcc 100644
--- a/arch/arm/cpu/armv7/zynq/Makefile
+++ b/arch/arm/cpu/armv7/zynq/Makefile
@@ -18,14 +18,10 @@ COBJS-y	+= ddrc.o
 COBJS-y	+= slcr.o
 COBJS-y	+= clk.o
 
-SOBJS-y	+= lowlevel_init.o
-
 COBJS	:= $(COBJS-y)
 
-SOBJS	:= $(SOBJS-y)
-
-SRCS	:= $(SOBJS:.o=.S) $(COBJS:.o=.c)
-OBJS	:= $(addprefix $(obj),$(SOBJS) $(COBJS))
+SRCS	:= $(COBJS:.o=.c)
+OBJS	:= $(addprefix $(obj),$(COBJS))
 
 all:	$(obj).depend $(LIB)
 
diff --git a/arch/arm/cpu/armv7/zynq/cpu.c b/arch/arm/cpu/armv7/zynq/cpu.c
index cd96862..6bdbe87 100644
--- a/arch/arm/cpu/armv7/zynq/cpu.c
+++ b/arch/arm/cpu/armv7/zynq/cpu.c
@@ -13,6 +13,10 @@
 #define ZYNQ_SILICON_VER_MASK	0xF0000000
 #define ZYNQ_SILICON_VER_SHIFT	28
 
+void lowlevel_init(void)
+{
+}
+
 int arch_cpu_init(void)
 {
 	zynq_slcr_unlock();
diff --git a/arch/arm/cpu/armv7/zynq/lowlevel_init.S b/arch/arm/cpu/armv7/zynq/lowlevel_init.S
deleted file mode 100644
index 6d714b7..0000000
--- a/arch/arm/cpu/armv7/zynq/lowlevel_init.S
+++ /dev/null
@@ -1,26 +0,0 @@
-/*
- * Copyright (C) 2013 Xilinx, Inc. All rights reserved.
- *
- * SPDX-License-Identifier:	GPL-2.0+
- */
-
-#include <asm-offsets.h>
-#include <config.h>
-#include <linux/linkage.h>
-
-ENTRY(lowlevel_init)
-
-	/* Enable the the VFP */
-	mrc	p15, 0, r1, c1, c0, 2
-	orr	r1, r1, #(0x3 << 20)
-	orr	r1, r1, #(0x3 << 20)
-	mcr	p15, 0, r1, c1, c0, 2
-	isb
-	fmrx	r1, FPEXC
-	orr	r1,r1, #(1<<30)
-	fmxr	FPEXC, r1
-
-	/* Move back to caller */
-	mov	pc, lr
-
-ENDPROC(lowlevel_init)
-- 
1.7.9.5

